frontend/src/components/core/Container.tsx

```diff
@@ -70,6 +74,12 @@ export const MainContainer = styled.div`
   width: 80%;
 `;
 
+export const TopContainer = styled.div`
```

Comment:
nit: rename this and the other TopX components to ProblemX (since for not it's only being used for those pages I think)

Reply:
Done! ✅